spec/rubocop/cop/style/struct_inheritance_spec.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::StructInheritance do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense when inheriting from Struct.new' do
+    inspect_source(cop,
+                   ['class Person < Struct.new(:first_name, :last_name)',
```

Comment:
@bbatsov there already is one or do you mean something else?: https://github.com/bbatsov/rubocop/pull/1571/files?diff=unified#diff-2acbcfc063c998c3c95c51657c537eebR26